android/SmartcarMqttController/app/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Sat Apr 30 13:11:29 CEST 2022
+sdk.dir=C\:\\Users\\drago\\AppData\\Local\\Android\\Sdk
```

Comment:
This file should not be version controlled. It should be removed and then added to gitignore